bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -78,7 +84,7 @@ impl WasmBezier {
 	}
 
 	pub fn length(&self) -> f64 {
-		self.0.length()
+		self.0.length(None)
 	}
 
 	pub fn compute(&self, t: f64) -> JsValue {
```

Comment:
What would you think about renaming `compute()` to `evaluate()`? To me, at least, it has a more intuitive meaning. But maybe that's just me.